### DISCUSSION TITLE
When naming the title of a new project, it says "Error, no response received"

### DISCUSSION THREAD

**Discussion by bueredlemon11** [asreview/asreview#1683] [Troubleshooting (report errors in the "Issues" tracker)]
Hello guys,
I'm new to ASReview, when I tried to create a new project, in the field "Title(required)", no matter what title I put, it always shows "Error, no response received". So I'm stuck at the first step...
Does anyone know why? Thanks in advance！

![Title error1](https://github.com/asreview/asreview/assets/131249758/9357cb6f-76ba-4b7d-a2a1-e77c0d7d139d)

**Comment by Rensvandeschoot** [comment]
What is the message you see in the Command Prompt? (the screen where you launched ASReview)?

**Reply by mojuha** [reply]
I don't understand the question :) but I have the same problem and this is what I read in my CMD:

python -m asreview lab
Serving ASReview LAB at http://localhost:5000/

If your browser doesn't open. Navigate to http://localhost:5000/

127.0.0.1 - - [2024-03-09 14:05:01] "GET / HTTP/1.1" 200 819 0.011831
127.0.0.1 - - [2024-03-09 14:05:01] "GET /static/js/main.3de5b682.js HTTP/1.1" 304 299 0.254736
127.0.0.1 - - [2024-03-09 14:05:01] "GET /static/css/main.9e747be3.css HTTP/1.1" 304 298 0.009281
127.0.0.1 - - [2024-03-09 14:05:01] "GET /boot HTTP/1.1" 200 257 0.000566
127.0.0.1 - - [2024-03-09 14:05:01] "GET /boot HTTP/1.1" 200 257 0.000000
127.0.0.1 - - [2024-03-09 14:05:01] "GET /api/projects/stats HTTP/1.1" 200 270 0.001085
127.0.0.1 - - [2024-03-09 14:05:01] "GET /api/projects HTTP/1.1" 200 228 0.000977
127.0.0.1 - - [2024-03-09 14:05:02] "GET /static/media/asreview_sub_logo_lab_black_transparent.c4e14af313ba97465fff113a6b2e4fd7.svg HTTP/1.1" 304 358 0.025774
127.0.0.1 - - [2024-03-09 14:05:02] "GET /static/media/roboto-latin-500.f5b74d7ffcdf85b9dd60.woff2 HTTP/1.1" 304 219 0.010217
127.0.0.1 - - [2024-03-09 14:05:02] "GET /static/media/ElasArrowRightAhead.e624bdd7c691c7c4f6c01604979c8f07.svg HTTP/1.1" 304 338 0.010801

**Reply by J535D165** [reply]
What is your operating system and version (Older version of MacOS?)

**Reply by mojuha** [reply]
Mine is Windows 10 Enterprise.

**Reply by mojuha** [reply]
Ok, now/today it works anyway. I don't know why but all good.

**Comment by Joanna40433322** [comment]
Do you know how to solve this problem, I also encountered this problem, thank you very much

**Reply by Rensvandeschoot** [reply]
What is the message in the CLI?

**Comment by esmee-r** [comment]
I have the same problem and can't continue with creating a project and therefore start screening.. Any solutions for this?
